app/controllers/healthcare_controller.rb

```diff
@@ -27,11 +27,17 @@ def summary
     render 'summary/healthcare'
   end
 
+  def confirm
+    workflow = DocumentWorkflow.new(healthcare)
+    workflow.update_status(:complete)
```

Comment:
`DocumentWorkflow#update_status` can return `false` if the new document status isn't possible. What kind of error handling do you think there should be here?